skyportal/tests/api/test_photometry.py

```diff
@@ -4,17 +4,21 @@
 from skyportal.tests import api
 from skyportal.models import Thumbnail, DBSession, Photometry
 
+import numpy as np
+import sncosmo
 
-def test_token_user_post_get_photometry_data(upload_data_token, public_source):
+
+def test_token_user_post_get_photometry_data(upload_data_token, public_source,
```

Comment:
Should be including tests for `ra` and `dec` (fail when not provided, pass when given as null, etc.).